rfcs/text/0013-multiple-files.md

```diff
@@ -0,0 +1,220 @@
+# 0013: Multiple files in an event
+
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **2 (Draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+Many security events refer to more than one file at the same time or contain duplicated attributes for files.
+Examples of this are file copies or modifications.
+ECS has current support for multiple users in an event that was originally introduced in a previous RFC, this builds on the ideas introduced there, but for files.
+
+## Fields
+
+The new fields discussed in this RFC are the following:
+
+- `file.target.*`
+- `file.changes.*`
+
+Note that `file` would need to be made reusable with the following entries.
+
+```YAML
+# schemas/file.yml excerpt
```

Comment:
Thanks for defining the field definitions already 👍 

Can you also add these in a separate, standalone YAML file? Similar to what was done [here](https://github.com/elastic/ecs/blob/master/rfcs/text/0007/user.yml)?

The standalone file allows for anyone to grab it independently using `--include` for testing the schema changes, e.g.:

`$ python scripts/generator.py --include rfcs/text/00xx/file.yml`